tasks/update-code/git.yml

```diff
@@ -1,6 +1,30 @@
 ---
+
+- name: ANSISTRANO | GIT | Ensure GIT deployment key is up to date
+  copy:
+    src: "{{ ansistrano_git_identity_key_path }}"
+    dest: ".ssh/git_identity_key"
```

Comment:
Is this path relative to the deployment directory (`{{ansistrano_deploy_to}}`) or to the user `$HOME` directory?